tensorflow/contrib/cudnn_rnn/python/kernel_tests/cudnn_rnn_ops_test.py

```diff
@@ -87,10 +88,14 @@ def RunLSTM(sess,
 
   shape = ([time, batch_size, input_size] if time_major else
            [batch_size, time, input_size])
-  inputs = variable_scope.get_variable(
+  inputs_np = np.random.rand(*shape).astype(dtype.as_numpy_dtype)
```

Comment:
nitpick: it's a minor suggestion, up to you.
No need to create both nodes in the graph
```py
inputs_np = ...
if dynamic_shape_input:
  inputs = get_variable(...)
else:
  inputs = placeholder(...)
```

In fact, I think maybe placeholder alone is good -- if it works for dynamic shape input, it must work for static shape.

Reply:
I thought this previously. But actually, I think we need to keep both nodes. I tried the dynamic inputs for rnn.dynamic_rnn() but it doesn't work. (I guess they cannot handle the loops dynamically?).

I can remove the static input tests if that is OK for you.